src/modules/CurrencyModule.cpp

```diff
@@ -914,21 +915,113 @@ CurrencyModule::CurrencyModule(UmikoBot* client) : Module("currency", true), m_c
 			client.createMessage(message.channelId(), "**I can't let you make your victim go into serious debt.**");
 			return;
 		}
+		QString thiefName = UmikoBot::Instance().GetName(channel.guildId(), authorId);
+		QString victimName = UmikoBot::Instance().GetName(channel.guildId(), victimId);
+
+		auto& victimCurrency = guildList[channel.guildId()][getUserIndex(channel.guildId(), victimId)];
+		auto& authorCurrency = guildList[channel.guildId()][getUserIndex(channel.guildId(), authorId)];
+
+		EventModule* eventModule = static_cast<EventModule*>(UmikoBot::Instance().GetModuleByName("event"));
+		auto& eventConfig = eventModule->getEventServerData(channel.guildId());
+		
+		if (eventConfig.eventHighRiskHighRewardRunning)
```

Comment:
When is this event enabled exactly?

Reply:
# HighRiskHighReward
**The steal chance is decreased, but if the users succeed on stealing then they will get BONUS CC**
- The BONUS is proportionate to the amount of the CC that a user steals.

Reply:
*When* is it enabled, though?

Reply:
I can't see where you change this boolean.

Reply:
Never mind, the diff was closed